### ISSUE TITLE
R-CNN

### ISSUE THREAD

**Issue by dev2y** [jeshraghian/snntorch#182]
Hi is there any training example for R-CNN? Thanks.

**Comment by dev2y** [comment]
got it. I could compile with gcc 12.1

**Comment by jeshraghian** [comment]
Not yet.

**Comment by dev2y** [comment]
@jeshraghian Thank you. But it is possible to do it. Right?

**Comment by jeshraghian** [comment]
Definitely! The best way to get started is to use a pre existing PyTorch implementation of R-CNN. Then between each Conv2d/Linear layer, include a `Leaky` neuron from snntorch. 

You can pass the spikes from the Leaky layer to the pytorch layers.

It might struggle to learn anything to start with, so you may need to play around with optimizers, surrogate gradients, and lowering firing thresholds. It'll be a little tedious, but the discord channel is the best place to get help.

**Comment by dev2y** [comment]
Thank you. I will give it a try.
